### ISSUE TITLE
Sathrah cleansing after quest is complete casts a spell. (it's supposed to be instant)

### ISSUE THREAD

**Issue by hgarner17** [Wall-core/Everlook-Bugtracker#419]
Just a simple qol inconvenience. In SoM and in Classic the spell to cleanse the spider was instant cast. You could drop down instant cast and run back up for reward with no delay.

The way it currently is there is a long casting spell. (5 seconds maybe?) It was not like this in either classic or SoM
![Sathrah's Sacrifice still casting a spell](https://user-images.githubusercontent.com/35550935/197377090-cb47a817-c881-487c-9a53-c816c61a9992.png)

**Comment by FlagFlayer** [comment]
Related to how the item is programmed in vmangos. You can submit a bug report at https://github.com/vmangos/core/issues

**Comment by Bornlesszero** [comment]
Some things from Vanilla pservs r diff from classic n som. This is intended to be like this on pservs

**Comment by FlagFlayer** [comment]
You are completely wrong. The main aim of vmangos as a project is to reproduce blizz-like vanilla as accurately as possible. If Classic (which uses blizzards vanilla data), behaves differently than current behaviour, the issue should be reported.
